plugins/llvm/llvm_binary.hpp

```diff
@@ -337,12 +363,13 @@ struct section {
 };
 
 std::vector<section> read(const ObjectFile* obj) {
-    int size = utils::distance(obj->begin_sections(),
-                               obj->end_sections());
+    int size = std::distance(obj->sections().begin(), 
```

Comment:
`std::distance` returns a type that is derived from parameters (usually it is `ptrdiff_t` that is likely may not fit into an `int`). So, to prevent unnecessary coercions use `auto` instead of int. Actually it is better to use `auto` whenever possible in modern C++.